guides/common/modules/ref_system-requirements.adoc

```diff
@@ -116,3 +117,13 @@ ISS Network Sync works across all {Project} versions that support it.
 For more information, see {ContentManagementDocURL}Synchronizing_Content_Between_Servers_content-management[Synchronizing Content Between {Project} Servers] in _{ContentManagementDocTitle}_.
 endif::[]
 endif::[]
+endif::[]
+
+ifeval::["{context}" == "quick-start"]
+The installation requires {project-minimum-memory} of memory.
+For more information, see {InstallingServerDocURL}system-requirements_{project-context}[System Requirements].
+
+The Foreman installer uses Puppet to install Foreman.
```

Comment:
I'm not sure this sentence really adds a lot. It came from the old quick start guide, but IMHO it's an implementation detail.
```suggestion
```